python/lsst/ts/salobj/base_script.py

```diff
@@ -587,6 +588,22 @@ def do_setCheckpoints(self, data):
             await self._exit()
 
     def set_checkpoints(self, *, pause, stop):
+        """Set the pause and stop checkpoint fields and output the event.
+
+        Parameters
+        ----------
+        pause : `string`
+            Checkpoint(s) at which to pause, as a regular expression.
+            "" to not pause at any checkpoint; "*" to pause at all checkpoints.
+        stop : `string`
+            Checkpoint(s) at which to stop, as a regular expression.
+            "" to not stop at any checkpoint; "*" to stop at all checkpoints.
```

Comment:
Should these both have `""` as default kwargs?

Reply:
I don't think so. This is intended to be a low-level API that will only be called when both values are being set. Oh heck, I'll make it private.